### PR TITLE
Set Tempo by typing number into dialog

### PR DESCRIPTION
**Pull request by georgkrause**
Right click on "Tap"-Button and type any number. Issues #198 Obiously numbers between 60 and 220 does not make any sense so i will introduce an check. 

Feedback welcome.

### CODE REVIEW

File: src/gmastertrack.cxx [hunk 1]
```diff
@@ -169,8 +169,15 @@ static void gmastertrack_button_callback(Fl_Widget *w, void *data)
 
 
 	} else if ( strcmp( w->label(), "Tap" ) == 0 ) {
-		EventTimeTempoTap e;
-		writeToDspRingbuffer( &e );
+		if ( Fl::event_button() ==FL_RIGHT_MOUSE) {
+			const char* answer = fl_input("Enter BPM value: ");
```
Comment (harryhaaren) [inline src/gmastertrack.cxx]: I belive this will crash in the exact same way you reported on the other thread? Try clicking cancel and see what happens...

Comment (harryhaaren) [reply src/gmastertrack.cxx]: Fixed and pushed, thanks

**Comment by georgkrause** [comment]
@harryhaaren you are right, but as mentioned it wasnt ready at all, that you merged this isnt that good, try to type any number <60 or >220. this makes crazy things on the GUI... 

Will file another PR with the test if the value is between 60 and 220.

**Comment by harryhaaren** [comment]
Sure, a patch to sanity check the value is welcome. Sorry if I merged to soon, -Harry

**Pull request closed without merging**